src/ReactART.js

```diff
@@ -261,7 +258,11 @@ var EventTypes = {
   onClick: 'click'
 };
 
-var NodeMixin = assign({}, ReactComponentMixin, {
+var NodeMixin = {
+
+  construct: function(element) {
+    this._currentElement = element;
```

Comment:
Previously, `construct` set `this._mountImage = null;`, right? Do we still need that?